### PR TITLE
fixed commits

### PR DESCRIPTION
**Pull request by tjquinn1**

### CODE REVIEW

**Comment by tjquinn1** [comment]
I didn't end up using poetry because I was having a bunch of issues. I think it is a local issue but didn't want to spend any more time on it.

File: place_resolver.py [hunk 1]
```diff
@@ -76,6 +77,58 @@ def filter_place_entries(self, entries):
...
+        se = coords[2]
+        dd = 0;
+        if d == 0:
+            dd = ((m / 60.0) + (se / 3600.0));
+        else:
+            # python doesn't have a clean way to get the symbol of a number so we use copysign
+            dd = math.copysign(1,d) * abs(d) + (m / 60.0) + (se / 3600.0)
+        return round(dd, 6)
+
+    def process_length_2_coords(coords):
```
Comment (jdamerow) [inline place_resolver.py]: where is this being used? should that be used in lines 112 and 123?

File: place_resolver.py [hunk 2]
```diff
@@ -76,6 +77,58 @@ def filter_place_entries(self, entries):
...
+        split_string =  coord.split("|")
+        coords = []
+        formatted_coords = []
+        for string in split_string:
+            try:
+                # check if string is a int/float and cast to float
+                coords.append(float(string))
+            except:
+                pass
+            if string in ["N", "E"]:
```
Comment (jdamerow) [inline place_resolver.py]: I take it that N, E, S, W are at the end of a coordinate string? an example in a comment/documentation would help to understand what this is doing.

**Review by jdamerow — CHANGES_REQUESTED** [review]

**Comment by jdamerow** [comment]
I get this exception:
```
Traceback (most recent call last):
  File "place_resolver.py", line 245, in <module>
    main()
  File "place_resolver.py", line 237, in main
    result = resolver.resolve_place(place["place_name"])
  File "place_resolver.py", line 152, in resolve_place
    'cleaned_coordinates': self.clean_coordinates(potential_place_entries[0]['coordinates'])
TypeError: clean_coordinates() takes 1 positional argument but 2 were given
```

**Comment by jdamerow** [comment]
closing due to coordinate bug

**Pull request merged**